app/Filament/Admin/Resources/Nodes/RelationManagers/ServersRelationManager.php

```
@@ -25,29 +30,27 @@ public function setTitle(): string
-            ->heading('')
+            ->heading(null)
             ->columns([
                 TextColumn::make('user.username')
                     ->label(trans('admin/node.table.owner'))
-                    ->url(fn (Server $server): string => route('filament.admin.resources.users.edit', ['record' => $server->user]))
+                    ->url(fn (Server $server) => user()?->can('update', $server->user) ? EditUser::getUrl(['record' => $server->user]) : null)
                     ->searchable(),
                 TextColumn::make('name')
                     ->label(trans('admin/node.table.name'))
-                    ->url(fn (Server $server): string => route('filament.admin.resources.servers.edit', ['record' => $server]))
+                    ->url(fn (Server $server) => user()?->can('update', $serve) ? EditServer::getUrl(['record' => $server]) : null)
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$serve'. (undefined)

(UnusedLocalVariable)